docs/userguide/keywords.rst

```diff
@@ -98,9 +98,9 @@ unless you need the associated ``setuptools`` feature.
     added to the tests to be run.  If the named suite is a package, any
     submodules and subpackages are recursively added to the overall test suite.
 
-    Specifying this argument enables use of the `test`_ command to run the
+    Specifying this argument enables use of the :ref:`test <test>` command to run the
```

Comment:
`<test>` is necessary to have `test` as an anchor and not the title that may be set in the other document under the label.